js/scripts.js

```diff
@@ -314,16 +335,23 @@ navList.onclick = () => {
  */
 const validateMsg = document.getElementById('validation-msg');
 const form = document.getElementById('contact-form');
+const messageObj = {};
 
 const submitForm = (e) => {
   const email = document.getElementById('email').value;
+  const fullName = document.getElementById('name').value;
+  const message = document.getElementById('message').value;
 
   if (email !== email.toLowerCase()) {
     e.preventDefault();
     validateMsg.innerText = 'Kindly enter email in lowercase';
     return false;
   }
 
+  messageObj.name = fullName;
+  messageObj.email = email;
+  messageObj.message = message;
+  localStorage.setItem('messageObject', JSON.stringify(messageObj));
```

Comment:
Form values are not being set if the page is reloaded before form submission because you save data on submission. The data object should be saved if the user changes the content of any input field so if they only entered their name they can see it after a page reload. Please make the necessary changes so that your form shows the saved data 😉